docs/ha-mode.md

```diff
@@ -15,7 +15,7 @@ The etcd clients (kube-api-masters) are configured with the list of all etcd pee
 ## Kube-apiserver
 
 K8s components require a loadbalancer to access the apiservers via a reverse
-proxy. Kubespray includes support for an nginx-based proxy that resides on each
+proxy. Kubespray includes support for a nginx-based proxy that resides on each
```

Comment:
The original seems good for me.

Reply:
ditto, "an nginx" seems like the better version